test/src/main/java/com/demoqa/MainPage.java

```diff
@@ -0,0 +1,33 @@
+package com.demoqa;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class MainPage extends BasePage {
+
+    private static final By ELEMENTS_CARD = By.xpath("//div[@class='card-body']/h5[text()='Elements']");
```

Comment:
Decide what kind of elements do you want in constants: `By` or `String`. And stick to it in all the classes